fintech-examples/fintech-ui/src/app/bank/common/transaction-card/transaction-card.component.html

```diff
@@ -10,7 +10,8 @@
         }}</span>
       </div>
       <div class="right">
-        <span class="ltx-amount-number">{{ transaction?.transactionAmount?.amount }}</span>
+        <span [ngClass]="{'text-danger': transaction?.transactionAmount?.amount < 0}"
```

Comment:
I suggest to set the directive to the div-tag (line 12) so that also the currency becomes red